api/mfx_dispatch/linux/mfxloader.cpp

```diff
@@ -442,7 +442,11 @@ mfxStatus MFXVideoUSER_Load(mfxSession session, const mfxPluginUID *uid, mfxU32
       if (MFX::g_GlobalCtx.m_plugins.empty()) {
         // Parsing plugin configuration file and loading information of
         // _all_ plugins registered on the system.
+#ifdef ANDROID
```

Comment:
I would try to avoid #ifdefs in the code as much as we can. We already have a lot. Thus, questions:
1. Why android defines some other name for plugins.cfg file? If you want to separate mediasdk config file from anything else, why you can't create mediasdk dedicated folder? For example: /vendor/etc/mediasdk/plugins.cfg
2. Why you can't pass /vendor/etc/mediasdk via MFX_PLUGINS_CONF_DIR?

Reply:
Agree, moved to Android.mk and replaced by plugins.cfg